src/App.jsx

```diff
@@ -1,33 +1,73 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from "react"
+import "./App.css"
+import Inbox from "./components/Inbox"
+import InboxDetails from "./components/InboxDetails"
+import NavBar from "./components/NavBar"
+import Search from "./components/Search"
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [emailData, setEmailData] = useState([])
+
+  useEffect(() => {
+    fetch("https://email-client-api.dev.io-academy.uk/emails")
+      .then((response) => response.json())
+      .then((data) => {
+        setEmailData(data.data)
+      })
+  }, [])
+
+  const [hiddenToggle, setHiddenToggle] = useState("hidden")
+
+  function handleClick() {
+    if (hiddenToggle === "hidden") {
+      setHiddenToggle("block")
+    } else {
+      setHiddenToggle("hidden")
+    }
+  }
 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
+      <div className="flex bg-gray-700 text-white items-center w-screen justify-between p-6">
+        <button
+          className="border p-2 rounded-md md:hidden"
+          onClick={handleClick}
+        >
+          Menu
         </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
+        <h1>Email Client</h1>
+        <div className="flex items-center">
+          <span className="md:pr-4 pr-2 text-2xl">
+            <i class="fa-solid fa-circle-user"></i>
+          </span>
+          <h4>User Name</h4>
+        </div>
+      </div>
+      <div className="flex w-screen">
+        <NavBar status={hiddenToggle} />
+        <div className="p-4 border md:w-2/12 min-w-10 w-screen md:static relative sm:z-0">
+          <Search />
+          <div className="overflow-y-auto max-h-screen">
+            {emailData.map((email) => (
+              <Inbox
```

Comment:
This isn't the Inbox? It is more like the Email Item? The div containing this component is the Inbox.